include/sdf/Light.hh

```diff
@@ -134,6 +134,14 @@ namespace sdf
     /// \param[in] _cast True to indicate that the light is on, False otherwise.
     public: void SetLightOn(const bool _isLightOn);
 
+    /// \brief Get if the visual light is enabled
```

Comment:
I think it's more common to refer to it as the "light visual", or maybe even "light visualization" / "light indicator" may be clearer.